packages/server/src/plugin/usageReporting/stats.ts

```diff
@@ -35,6 +35,8 @@ export class SizeEstimator {
   bytes = 0;
 }
 export class OurReport implements Required<IReport> {
+  tracesPreAggregated = false;
```

Comment:
I guess this is necessary because of the `Required` on the previous line?

Given that it's here, a comment along the lines of "Apollo Server includes each operation either as aggregated stats or as a trace, but not both. Other reporting agents such as Apollo Router include *all* operations in stats (even those that are sent as traces), and they set this flag to true." could be nice

Reply:
[fc449e5](https://github.com/apollographql/apollo-server/pull/7411/commits/fc449e5b2d4dcc1df8cbf2508294acbef8e2cf6a)